crypto/fipsmodule/ec/ec_test.cc

```diff
@@ -1036,6 +1037,15 @@ TEST(ECTest, ArbitraryCurve) {
   EXPECT_EQ(0, EC_GROUP_cmp(group.get(), group.get(), NULL));
   EXPECT_EQ(0, EC_GROUP_cmp(group2.get(), group.get(), NULL));
 
+  bssl::UniquePtr<BIGNUM> converted_generator1(EC_POINT_point2bn(
+      group.get(), generator.get(), POINT_CONVERSION_UNCOMPRESSED, NULL, NULL));
```

Comment:
An elliptic curve point can be encoded as a compressed point as well. The enum is `POINT_CONVERSION_COMPRESSED`. Worth doing that path as well?